test/clojure/core_test/some_fn.cljc

```diff
@@ -0,0 +1,46 @@
+(ns clojure.core-test.some-fn
+  (:require [clojure.test :refer [deftest testing is are]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer) [when-var-exists]]))
+
+(when-var-exists some-fn
+  (deftest test-some-fn
```

Comment:
I think there's a balance we want to find in testing. Our goal for this file is to test `some-fn`, but we also end up testing:

1. That sets are callable
2. That maps are callable
3. That keywords are callable

What we're really looking for there is:

1. `some-fn` accepts both `fn?` values and `ifn?` values

In terms of `some-fn` itself, that's what matters. So I think we have some over testing here which can be removed, so we can just focus on what `some-fn` can do.

Reply:
Thanks for the explanation, added `testing "supports ifn"`.